ush/global_ens/evs_get_gens_atmos_data.sh

```diff
@@ -478,18 +527,45 @@ if [ $modnam = gefs_apcp24h ] ; then
      for mb in 01 02 03 04 05 06 07 08 09 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 ; do
         typeset -a lead_arr
         for lead_chk in 024 036 048 060 072 084 096 108 120 132 144 156 168 180 192 204 216 228 240 252 264 276 288 300 312 324 336 348 360 372 384; do
-         if [ -s $COMOUTgefs/gefs.ens${mb}.t${ihour}z.grid3.f${lead_chk}.grib2 ] ; then 	
+          file1=gefs.ens${mb}.t${ihour}z.grid3.f${lead_chk}.grib2
+          echo $file1
+          if [ $lead_chk -ge 108 ]; then
+             file2=gefs.ens${mb}.t${ihour}z.grid3.f$(printf '%03d' $((lead_chk-6))).grib2
+             file3=gefs.ens${mb}.t${ihour}z.grid3.f$(printf '%03d' $((lead_chk-12))).grib2
+             file4=gefs.ens${mb}.t${ihour}z.grid3.f$(printf '%03d' $((lead_chk-18))).grib2
+             echo "file2: $file2"
+             echo "file3: $file3"
+             echo "file4: $file4"
+          else
+             strip_lead_chk=$(echo $lead_chk | sed 's/^0*//')
+             file2=gefs.ens${mb}.t${ihour}z.grid3.f$(printf '%03d' $((strip_lead_chk-6))).grib2
+             file3=gefs.ens${mb}.t${ihour}z.grid3.f$(printf '%03d' $((strip_lead_chk-12))).grib2
+             file4=gefs.ens${mb}.t${ihour}z.grid3.f$(printf '%03d' $((strip_lead_chk-18))).grib2
+             echo "file2: $file2"
+             echo "file3: $file3"
+             echo "file4: $file4"
```

Comment:
Lines 536-538 and 544-546 can be removed. They were included for diagnostics and troubleshooting, but are not needed for the final code.

Reply:
See below for the most recent push.